src/actors/mod.rs

```diff
@@ -0,0 +1,74 @@
+use crate::settings::toml::{ActorNamespace, ActorNamespaceNoId};
+use reqwest::blocking::Client;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ApiActorNamespace {
+    name: String,
+    id: String,
+}
+
+#[derive(Deserialize)]
+pub struct ListActorNamespaceResponse {
+    result: Option<Vec<ApiActorNamespace>>,
+}
+
+pub fn get_existing_namespaces(
+    client: &Client,
+    account_id: &str,
+) -> Result<HashMap<String, String>, failure::Error> {
+    let mut map = HashMap::new();
+    let namespaces = list_namespaces(client, account_id)?;
+    for namespace in namespaces {
+        map.insert(namespace.name, namespace.id);
+    }
+
+    Ok(map)
+}
+
+pub fn convert_namespace_names_to_ids(
+    client: &Client,
+    account_id: &str,
+    namespaces_noid: &Vec<ActorNamespaceNoId>,
+) -> Result<Vec<ActorNamespace>, failure::Error> {
+    let namespaces = get_existing_namespaces(client, account_id)?;
+    namespaces_noid.iter().map(|ns| {
+        if let Some(id) = namespaces.get(&ns.name) {
+            Ok(ActorNamespace {
+                binding: ns.binding.clone(),
+                id: id.clone()
+            })
+        } else {
+            failure::bail!("actor namespace binding {} is configured with {}, a namespace that does not exist", ns.binding, ns.name);
```

Comment:
I'd love to give an error if the binding name here is also in the implements block... but that means making it accessible here :/